components/match2/wikis/warcraft/match_legacy.lua

```diff
@@ -0,0 +1,281 @@
+---
+-- @Liquipedia
+-- wiki=warcraft
+-- page=Module:Match/Legacy
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local MatchLegacy = {}
+
+local Array = require('Module:Array')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Namespace = require('Module:Namespace')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper', {requireDevIfEnabled = true})
+
+local Opponent = require('Module:OpponentLibraries').Opponent
+
+local UNKNOWNREASON_DEFAULT_LOSS = 'L'
+local TBD = 'TBD'
+
+---@param match table
+---@param options {storeMatch2:boolean, storeMatch1:boolean, storePageVar:boolean, bracketId: string}
+---@return nil
```

Comment:
```suggestion
```
Unnecessary right?

Reply:
getting anno warnings without it:
![Screenshot 2023-10-13 11 04 31](https://github.com/Liquipedia/Lua-Modules/assets/75081997/fbba2f96-d46b-4003-9795-147ca1e8d3d6)

Reply:
Split the call and return then :)